.github/workflows/find-inactive-tsc.yml

```
@@ -0,0 +1,46 @@
+name: Find inactive TSC members
+
+on:
+  schedule:
+    # Run every Tuesday 12:05 AM UTC.
+    - cron: '5 0 * * 2'
+
+  workflow_dispatch:
+
+env:
+  NODE_VERSION: 16.x
```

Shouldn't this match other workflows?
```suggestion
  NODE_VERSION: lts/*
```

---

This dates to when 16.x was Current rather than LTS. The scripts needed 16.x. (I forget why.) But now that 16.x is LTS, I'll update both this and find-inactive-collaborators.yml. Thanks!